.gitignore

```diff
@@ -62,7 +62,7 @@ cover/
 .ruff_cache/
 
 # Django stuff:
-*.log
+*.log*
```

Comment:
what additional file types is this used for?

Reply:
the `TimedRotatingFileHandler` adds `.date` to the end of backups

Reply:
ah nice